generators/app/templates/babelrc_tmpl

```diff
@@ -0,0 +1,15 @@
+{
+  "presets": [
+    "env",
+    "react"
+  ],
+  "plugins": [
+    "transform-runtime",
+    [
+      "transform-es2015-classes",
```

Comment:
I thought we were getting rid of the es2015 stuff.

Reply:
Can you remind me please why we were removing the transform-es2015-classes plugin?

Reply:
@ThinkTankShark As far as I know these should not be needed when we are using the recommended "env" and "react" presets for babel. I have been removing them from the ship-components I changed recently. Do you have a specific reason for keeping them? 

@isuttell Please let me know if you think these transform plugins can be safely removed after migrating babel to use the env & react presets.

Reply:
@isuttell @JaredBoone Removed the plugins.